tests/e2e/index.ts

```diff
@@ -6,37 +6,37 @@ export * from './TestConstants';
 export * from './driver/ChromeDriver';
 export * from './driver/IDriver';
 export * from './driver/ContainerInitializer';
+export * from './utils/ScreenCatcher';
 export * from './utils/Logger';
-export * from './utils/workspace/TestWorkspaceUtil';
+export * from './utils/DriverHelper';
+export * from './utils/NameGenerator';
 export * from './utils/workspace/ITestWorkspaceUtil';
 export * from './utils/workspace/WorkspaceStatus';
-export * from './utils/NameGenerator';
```

Comment:
So many changes in index.ts. Are they all really needed?

Reply:
This is automatical changes, ignore them

Reply:
I would revert redundant changes at all to make PR clear to understand.

Reply:
Let's agree that this is not important